README.md

```diff
@@ -87,11 +87,20 @@ Accessibility should be incorporated and tested throughout the design and develo
 
 ## Policy & Informal
 
-* **Analytics:** All of our sites should have code snippets for 18F and Digital Analytics Program Google Analytics as well as Google Webmaster Tools.
 * Agency or Organization Communications Office
 * Legislative Affairs
 * Office of General Counsel
 
+### Analytics
+
+All of our sites should have code snippets for
+
+* 18F Google Analytics (18F only)
+* Digital Analytics Program (DAP)
+* Google Webmaster Tools
```

Comment:
What code snippet is required for GWT? The `<meta>` tag they offer you? I think it's better to say that the site should be approved in Google Webmaster Tools under your work Google Account, and team members should be shared as owners on the profile.